samples/net/cloud/aws_mqtt/src/config.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __CONFIG_H__
+#define __CONFIG_H__
+
+#define ALIVE_TIME      (MSEC_PER_SEC * 60U)
+#define APP_MQTT_BUFFER_SIZE    128
+#error "Fill the Host Name and compile"
+#define CONFIG_AWS_HOSTNAME ""
```

Comment:
Hostname must be provided via Kconfig file in this case. A sample app can have a Kconfig file of its own, see other networking samples how to do it.
Probably also port and mqtt client id should be put to kconfig file too as these can change for different users.